README.md

```diff
@@ -2,3 +2,18 @@
 ![rust workflow](https://github.com/helium/proto/actions/workflows/rust.yml/badge.svg)
 ![node.js workflow](https://github.com/helium/proto/actions/workflows/node.yml/badge.svg)
 ![erlang workflow](https://github.com/helium/proto/actions/workflows/erlang.yml/badge.svg)
+
+## Contributing
+
+- Avoid `float` in Protobufs because representations differ across hardware architectures
+  + There are many floating point representations from IEEE, plus others
+  + i.e., some range of interior digits are random per float spec
+- Frequency should use `uint32`
+  + Locally, define accessor functions to return float after appropriate division
+- rssi or signal is always negative, thus use `sint32`
+- snr is signal-to-noise ratio and should be `uint32`
```

Comment:
We need to add level of precision for this ratio? or is it always a whole multiplier

Reply:
I think signal-to-noise ratio should be sint32 and in units of deci-db. I believe LoRa radios can actually receive below the noise floor.

Reply:
doesn't matter for the type instruction here though